ted2zim/entrypoint.py

```diff
@@ -75,19 +74,17 @@ def main():
 
     parser.add_argument(
         "--title",
-        help="Custom title for your project and ZIM. Default to Channel name (of first video if playlists)",
-        required=True,
+        help="Custom title for your project and ZIM. Default value - TED Collection",
+        default="TED Collection",
     )
 
     parser.add_argument(
         "--description",
-        help="Custom description for your project and ZIM. Default to Channel name (of first video if playlists)",
-        required=True,
+        help="Custom description for your project and ZIM. Default value - A selection of several topics' videos from TED",
+        default="A selection of several topics' videos from TED",
```

Comment:
Same as title

Reply:
Done. Now we update description if not provided and consider number of topics that have valid videos.